app/tests/integration/nimbus/conftest.py

```diff
@@ -143,10 +143,51 @@ def experiment_name(request):
     return f"{request.node.name[:75]}-{str(uuid.uuid4())[:4]}"
 
 
+@pytest.fixture(name="load_experiment_outcomes")
+def fixture_load_experiment_outcomes():
+    """Fixture to create a list of outcomes based on the current configs."""
+    outcomes = {"firefox_desktop": "", "fenix": "", "firefox_ios": ""}
+    base_path = "/code/app/experimenter/outcomes/jetstream-config-main/outcomes"
+
+    for k in list(outcomes):
+        outcomes[k] = [
+            name.split("_")[0].rsplit(".")[0]
+            for name in os.listdir(f"{base_path}/{k}")
+            if "example" not in name
+        ]
+    return outcomes
+
+
 @pytest.fixture
-def default_data(application, experiment_name):
+def default_data(application, experiment_name, load_experiment_outcomes):
     feature_config_id = APPLICATION_FEATURE_IDS[application]
 
+    outcomes = {
+        "firefox_desktop": BaseExperimentMetricsDataClass(
+            primary_outcomes=[load_experiment_outcomes["firefox_desktop"][0]],
+            secondary_outcomes=[load_experiment_outcomes["firefox_desktop"][1]],
+        ),
+        "fenix": BaseExperimentMetricsDataClass(
+            primary_outcomes=[load_experiment_outcomes["fenix"][0]],
+            secondary_outcomes=[load_experiment_outcomes["fenix"][1]],
+        ),
+        "ios": BaseExperimentMetricsDataClass(
+            primary_outcomes=[load_experiment_outcomes["firefox_ios"][0]],
+            secondary_outcomes=[load_experiment_outcomes["firefox_ios"][1]],
+        ),
+    }
+
+    application_str = str(application).lower()
+
+    for _ in list(outcomes):
+        if _ in application_str and "focus" not in application_str:
+            metrics_data = outcomes[_]
+            break
```

Comment:
Shouldn't this just be

```py
if application_str in outcomes:
    metrics_data = outcomes[application_str]
else:
    ...
```

?

Reply:
Or you could add the empty entry to `focus` in the dict so then it's just

```py
metrics_data = outcomes[application_str]
```

Reply:
> Shouldn't this just be
> 
> ```python
> if application_str in outcomes:
>     metrics_data = outcomes[application_str]
> else:
>     ...
> ```
> 
> ?

Ah yes, I added an extra step. Reminds me to not code while sick

Reply:
Brain 🧠 not found 😴  Me toooooooooo 😓